data/translations/spanish/register.toml

```diff
@@ -0,0 +1,26 @@
+heading = "Inscríbase para votar"
+
+[by_mail]
+body = "Los residentes de %state_name% pueden inscribirse para votar por correo."
+download = "Descargar el formulario (PDF)"
+download_others = "Descargar el formulario en otros idiomas:"
+step_one_label = "Paso 1"
+step_one = "Llenar su registro de votante."
+step_two_label = "Paso 2"
+step_two = "Enviarla por correo a su estado."
+step_three_label = "Paso 3"
+step_three = "Votar en las próximas elecciones!"
```

Comment:
Some of these translations I tackled on my own, @arantxa. So your 👀 / contributions are greatly appreciated! Thanks!